packages/sdk/src/dataProtector/sharing/removeProtectedDataAsRentable.ts

```diff
@@ -0,0 +1,63 @@
+import { WorkflowError } from '../../utils/errors.js';
+import {
+  isCollectionOwner,
+  isProtectedDataInCollection,
+} from '../../utils/sharing.js';
+import { throwIfMissing } from '../../utils/validators.js';
+import {
+  IExecConsumer,
+  RemoveProtectedDataAsRentableParams,
+  RemoveProtectedDataAsRentableResponse,
+  SubgraphConsumer,
+} from '../types.js';
+import { getSharingContract } from './smartContract/getSharingContract.js';
+
+export const removeProtectedDataFromRenting = async ({
+  iexec = throwIfMissing(),
+  graphQLClient = throwIfMissing(),
+  collectionTokenId = throwIfMissing(),
+  protectedDataAddress = throwIfMissing(),
+}: IExecConsumer &
+  SubgraphConsumer &
+  RemoveProtectedDataAsRentableParams): Promise<RemoveProtectedDataAsRentableResponse> => {
+  //TODO:Input validation
+
+  const userAddress = await iexec.wallet.getAddress();
+  if (
+    !(await isCollectionOwner({
+      graphQLClient,
+      collectionId: collectionTokenId,
+      walletAddress: userAddress,
+    }))
```

Comment:
I would suggest to extract the `await` outside the `if`, for readability.
Same for next one on line 39.

Same for `setProtectedDataAsRentable.ts`